src/scalaDebugger.ts

```diff
@@ -75,7 +128,7 @@ class ScalaMainConfigProvider implements vscode.DebugConfigurationProvider {
         path: editor.document.uri.toString(true),
         runType: RunType.RunOrTestFile,
       };
-      return start(false, args).then(() => {
+      return start(debugConfiguration.noDebug, args).then(() => {
```

Comment:
Turns out that debugConfiguration contains the noDebug flag as a undeclared field. This is useful when running with F5 (Run without debugging)